tox.ini

```diff
@@ -0,0 +1,27 @@
+[flake8]
+max-line-length = 100
+
+[tox]
+skipsdist = True
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}/src:{toxinidir}/lib
+deps =
+    -rtest-requirements.txt
+    -rrequirements.txt
+
+[testenv:unit]
+commands =
+    pytest -vv tests/unit
+
+[testenv:lint]
+commands =
+    flake8 {toxinidir}/src {toxinidir}/tests
+    black --check {toxinidir}/src {toxinidir}/tests
+
+[testenv:integration]
+deps =
+    {[testenv]deps}
+    pytest-operator
+commands = pytest -v --tb native --show-capture=no --log-cli-level=INFO -s {posargs} {toxinidir}/tests/integration
```

Comment:
We should remove the `--tb native --show-capture=no` bits here, and rely on setting those via `{posargs}` if someone really wants to hide that for themselves. In CI, it's quite useful to show that extra information when something fails.